rwlib/CMakeLists.txt

```diff
@@ -41,10 +41,9 @@ SET(RWLIB_SOURCES
 	"source/job/WorkContext.cpp"
 	)
 
-add_library(rwlib
+set_source_files_properties(source/gl/gl_core_3_3.c PROPERTIES COMPILE_FLAGS -w)
```

Comment:
https://bitbucket.org/alfonse/glloadgen/issues/82/compiling-warning

So we'll need

```
extern "C" {
#include "foo.h"
}
```

if we don't have it yet + compile the file differently (different std flags).

I'm sure there is a proper solution for this issue, so I don't feel like settling for less.